worker/src/features/evaluation/evalService.ts

```diff
@@ -53,24 +58,17 @@ const getS3StorageServiceClient = (bucketName: string): S3StorageService => {
   return s3StorageServiceClient;
 };
 
-// this function is used to determine which eval jobs to create for a given trace
-// there might be multiple eval jobs to create for a single trace
-export const createEvalJobs = async ({
+const query = kyselyPrisma.$kysely.selectFrom("job_configurations").selectAll();
+
+type JobConfigurationsRaw = InferResult<typeof query>;
```

Comment:
Nit: can we take this type (https://github.com/langfuse/langfuse/blob/7d753a0930451e8c79e372e4d436e7b0a68c962b/packages/shared/prisma/generated/types.ts#L270)